app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -856,6 +857,9 @@ def rbac_field_changed(rec_type)
         # don't do anything to lookup box when checkboxes on the right side are checked
         page << set_element_visible('group_lookup', @edit[:new][:lookup]) unless params[:check]
       end
+      if rec_type == 'group'
+        page << changed ? 'ManageIQ.changes = true;' : 'ManageIQ.changes = false;'
```

Comment:
@bmclaughlin why do we have to set value of changed based upon ManageIQ.changes here, doesn't https://github.com/bmclaughlin/manageiq-ui-classic/blob/a969e99f7011e78fd1996ea669572fb0d5e04dcc/app/controllers/ops_controller/ops_rbac.rb#L826 work in this case?